lib/core/src/Cardano/Byron/Codec/Cbor.hs

```diff
@@ -462,10 +477,10 @@ decodeSharesProof = do
     _ <- CBOR.decodeBytes -- Vss Certificates Hash
     return ()
 
-decodeSlotId :: CBOR.Decoder s (Word64, Word32)
+decodeSlotId :: CBOR.Decoder s (EpochNo, Word32)
```

Comment:
For consistency, I'd suggest to also make `Word32` a `SlotNo` here.

Reply:
> For consistency, I'd suggest to also make `Word32` a `SlotNo` here.

Done.